src/frontend/packages/lib_video/tsconfig.json

```diff
@@ -10,12 +10,11 @@
     "noUnusedLocals": false,
     "noUnusedParameters": false,
     "declaration": true,
-    "declarationDir": "lib/",
+    "declarationDir": "lib/dts",
     "rootDir": "./src",
     "baseUrl": "./src",
     "paths": {
-      "altamoon-robust-websocket": ["types/libs/altamoon-robust-websocket"],
-      "vtt.js": ["types/libs/vtt.js"]
```

Comment:
I have a doubt here, it seems not necessary to add the path to `vtt.js`, I removed it because it was creating a bug after built with a path in `lib`.